test/e2e/e2e.go

```diff
@@ -29,25 +30,57 @@ import (
 
 var cfg = framework.DefaultConfig
 
+var isPrimary = false
+
 var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	addon.InitGlobals(cfg)
 
-	err := addon.ProvisionGlobals(cfg)
+	isPrimary = true
+
+	// We first setup the global addons, but do not provision them yet.
+	// This is because we need to transfer the data from the primary
```

Comment:
Explain what you mean by primary node so that it would make sense to a reader who might not remember exactly what Ginkgo does concurrently and where to look to find how we configure it.